### PR TITLE
Update the game over messages

### PR DESCRIPTION
**Pull request by knarf313**
Update the game over message so people know how to play again!

### CODE REVIEW

**Review by ZuyderlandFrank — COMMENTED** [review]
** Review **

I will check the code for issues

File: .github/steps/-step.txt
Comment (ZuyderlandFrank) [inline .github/steps/-step.txt:1]: Why +3 ???

**Review by ZuyderlandFrank — COMMENTED** [review]
Looks oaky

**Review by knarf313 — COMMENTED** [review]
okay for me

**Review by knarf313 — COMMENTED** [review]
Comment on the change

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
```
Comment (knarf313) [inline index.html:49]: `remove the icons from the h2-tag
```

Comment (knarf313) [inline index.html:49]: MySuggestion
```

File: index.html [hunk 2]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
...
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (knarf313) [inline index.html]:
```suggestion
<h2 hidden>Game over, refresh to play again!</h2>
```

**Pull request merged**